src/Legend/index.js

```diff
@@ -29,11 +29,13 @@ const Legend = ({
   acronym,
   children,
   color,
+  format,
   hideLabel,
   textColor,
   theme,
 }) => {
   const labelClasses = cx(
+    format,
```

Comment:
```suggestion
 theme[format],
```
aqui seria pega `theme.uppercase` ou `theme.capitalize`certo?

Reply:
Sim, corrigido! :+1: